src/main/java/org/spongepowered/api/tag/Taggable.java

```diff
@@ -24,37 +24,35 @@
  */
 package org.spongepowered.api.tag;
 
-import org.spongepowered.api.registry.DefaultedRegistryType;
 import org.spongepowered.api.registry.DefaultedRegistryValue;
-import org.spongepowered.api.registry.RegistryType;
 
 import java.util.Collection;
 
 /**
- * A type that may be included in one or more {@link Tag} collections.
+ * A {@link DefaultedRegistryValue} that may be included in one or more {@link Tag} collections.
```

Comment:
This feels like implementation creep onto the API where taggables must be defaulted registry values.